elide-integration-tests/src/test/java/com/yahoo/elide/tests/FilterIT.java

```diff
@@ -1654,6 +1655,146 @@ void testExceptionOnEmptyOperator() throws IOException {
 
     }
 
+    @Test
+    @Tag("memberOfOperation")
+    void testMemberOfOnAttributes() throws IOException {
+        JsonNode result;
+        String filterString = "Booker Prize";
+        Set<JsonNode> awardBook = new HashSet<>();
+        Set<JsonNode> nullNedAwardBook = new HashSet<>();
+
+
+        // * Filter On Root Entity *
+        for (JsonNode book : books.get("data")) {
+            Iterator<JsonNode> awards = book.get("attributes").get("awards").elements();
+            while (awards.hasNext()) {
+                if (awards.next().asText().equals(filterString)) {
+                    awardBook.add(book.get("id"));
+                    break;
+                }
+            }
+        }
+        // Test Default filter type on Root Entity
+        result = getAsNode(String.format("/book?filter[book.awards][hasmember]=%s", filterString));
+        assertEquals(awardBook.size(), result.get("data").size());
+        for (JsonNode book : result.get("data")) {
+            assertTrue(awardBook.contains(book.get("id")));
```

Comment:
There is a much cleaner way to do these comparisons with hamcrest Matchers.  There are some good examples in PaginationIT.